src/App.tsx

```diff
@@ -1,9 +1,30 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
+import moviesFromServer from './api/movies.json';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
-import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const getFilteredMovies = () => {
+    if (!query) {
+      return moviesFromServer;
+    }
+
+    const lowerCaseQuery = query.toLowerCase().trim();
+
+    return moviesFromServer.filter(movie => (
+      movie.title.toLowerCase().includes(lowerCaseQuery)
```

Comment:
move this checking into function and apply to both, title and description

Reply:
i had to replace the brach as I was using a new device and had to clone the task again

please check this commit:

9258bfaa7ad1a24f9965714ab0a005f97358d37b